public/index.html

```diff
@@ -1,13 +1,39 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie Server!</h1>
+<head>
+  <meta charset="utf-8">
+  <title>Hoodie</title>
+</head>
+<body>
+<h1>Hello from Hoodie Server!</h1>
 
-    <p>You see this because your app has no <code>public/</code> folder.</p>
-    <p>We are working on putting something more useful in here :)
-  </body>
+<p>You see this because your app has no <code>public/</code> folder.</p>
+<p> But don't worry, we're here to get you started! </p>
+<h2>Get going with Hoodie</h2>
+<p>Your first step should be creating a <code>public</code> folder inside your project folder.</p>
+<p>Next up, just create your index.html inside that public directory.</p>
+<p>We're almost here: Paste this into your index.html just before your closing tag for the body:
+  <strong> &lt;script src=hoodie/client.js"&gt;&lt;script&gt;</strong></p>
+<p>This is necessary for your own javascript files to pick up the hoodie methods.</p>
+<p>Now handle it just like you would handle your frontend project, hoodie won't get in your way. </p>
+<p>Setup your asset folders and create your .css and .js files.</p>
+<p>Happy coding :)</p>
+<h2>List of Hoodie methods</h2>
+<ul style="display:inline-block;list-style-type: none;">
+  <li style="font-size:115%;">Hoodie Store Methods</li>
+  <li>hoodie.store.find()</li>
+  <li>hoodie.store.findOrAdd()</li>
+  <li>hoodie.store.findAll()</li>
+  <li>hoodie.store.findAll()</li>
+
+</ul>
+<ul style="display:inline-block;list-style-type: none;">
+  <li style="list-style-type: none;font-size:115%;">Hoodie Account Methods</li>
+  <li>hoodie.store.add()</li>
+  <li>hoodie.store.find()</li>
+  <li>hoodie.store.findOrAdd()</li>
+  <li>hoodie.store.findAll()</li>
+
+</ul>
```

Comment:
We should either make the lists more helpful, or remove the entire `List of Hoodie methods` block at the end